src/main/java/org/openstreetmap/atlas/geography/sharding/SlippyTile.java

```diff
@@ -182,6 +182,49 @@ public Rectangle bounds()
         return this.bounds;
     }
 
+    @Override
+    public int compareTo(final SlippyTile other)
```

Comment:
Where is that comparison used?

Reply:
It's not used at this time, I was planning on using it when storing the processed Shards, but ended up not doing it. Might still prove to be useful so left it in.

Reply:
Ok, thanks!